schemas/json/widget/widget.schema.v1.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$id": "https://altinncdn.no/schemas/json/widget/widget.schema.v1.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Altinn Studio widget",
+    "description": "Schema that describes a widget for use in Altinn Studio",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+      "components": {
+        "$ref": "#/definitions/components"
+      },
+      "displayName": {
+        "$ref": "#/definitions/displayName"
+      },
+      "texts": {
+        "$ref": "#/definitions/texts"
+      }
+    },
+    "required": ["components", "displayName"],
+    "definitions": {
+      "components": {
+        "title": "The layout components associated with the widget",
+        "description": "Contains data describing the components that are part of the widget",
+        "type": "array",
+        "items": {
+          "$ref": "https://altinncdn.no/schemas/json/layout/layout.schema.v1.json#/definitions/component"
```

Comment:
Brilliant! 👍

Reply:
Hehe, I was pretty happy with this one 😄